src/test/java/uk/gov/ons/census/casesvc/testutil/DataUtils.java

```diff
@@ -91,6 +91,23 @@ public static ResponseManagementEvent getTestResponseManagementReceiptEvent() {
     return managementEvent;
   }
 
+  public static ResponseManagementEvent getTestResponseManagementSurveyLaunchedEvent() {
+    ResponseManagementEvent managementEvent = getTestResponseManagementEvent();
+
+    EventDTO event = managementEvent.getEvent();
+    event.setType(EventTypeDTO.SURVEY_LAUNCHED);
+    event.setSource("CONTACT_CENTRE_API");
+    event.setChannel("CC");
+
+    PayloadDTO payload = managementEvent.getPayload();
+    payload.setUac(null);
+    payload.setCollectionCase(null);
+    payload.setRefusal(null);
+    payload.setPrintCaseSelected(null);
```

Comment:
this payload will still contain a receipt, which we don't really want

Reply:
Changed to Response.